user-stories/US-07-Privacy-request-capture-service.md

```diff
@@ -0,0 +1,300 @@
+# Privacy Request capture service
+
+<!-- prettier-ignore -->
+| User Stories | 07 |
+| ---------- | ---- |
+| **Status** | Draft |
+| **Design** | [Figma activity diagram (draft)](https://www.figma.com/file/miUd9PEmLrjut53rwrQViX/Data-rights-request-capture-service?node-id=0%3A1)
+| **Previous discussions** | https://github.com/blindnet-io/product-ideas/issues/22
+| **Other documents** | [Privacy Request Interchange Vocabulary](../refs/schemas/priv/RFC-PRIV.md), [Examples of Privacy Requests](../refs/schemas/priv/examples.md), [Scenarios](../refs/schemas/priv/scenarios.md), [Expected Behavior of Systems](../refs/schemas/priv/expected-behavior.md)
+
+## Introduction
+
+Privacy Request Capture Service is a component of the [High Level Architecture](../refs/high-level-architecture). The user stories listed here illustrate its behavior, but are not to be considered exhaustive with regards to all the functionalities this component SHOULD offer.
```

Comment:
The idea of this mention is excalty to avoid that. In fact, there is a set of normative documents (HLC, HLA, etc.) only one of which is this US document.

The fact that something is missing from the US (and celarly present in HLA or HLC) does not mean it is to be ommitted from implementation.

Reply:
If your understanding of User Stories is that : 
`it must be exhaustive from the point of view of end-users interacting with the Privacy Request Capture Interface and wrt what that component offers to them` 

then:
- OK to remove this line
- My suggestion is to make this very clear in [guidelines](https://github.com/blindnet-io/product-management/blob/main/specifications/user-stories/README.md). If it was not clear to me, there might be other who do not understand it as such
- Please note my most vivid concern that:
    - it might be very difficult (if at all possible) to garantee exchaustiveness of the User Stories for this compoment at this point in time, i.e. prior to completing PRIV, and prior to deling with issues of Privacy Request Trasnfer (that is likely to impact functionning of this compoment) - i.e. "Define Privacy Request Multicast Protocol" in [issue 33, planned for Q3](https://github.com/blindnet-io/product-ideas/issues/33)
    - it might be suboptimal to wait for this document to become exhaustive (pending clarity on abovementioned issues) prior to pursuing the development of the related component.
- In particular, I consider the curently written User Stories to, most likely, not be exhaustive